bot/env.yaml

```diff
@@ -108,9 +112,6 @@ REINSTALL_APP_BEFORE_EACH_TASK: false
 # necessary to reproduce most crashes.
 REQUIRED_APP_ARGS: ''
 
-# Default quarantine size for ASan.
```

Comment:
We already support this via ADDITIONAL_*SAN_OPTIONS, so this is bad and creates mess due to different quarantine default on Android(16) vs non-Android (256)